movies/src/resources/context.jsonld

```diff
@@ -1,6 +1,16 @@
 {
   "@context": {
+    "mov": "https://hydra-movies.herokuapp.com/doc#",
     "hydra": "http://www.w3.org/ns/hydra/core#",
+    "hydra:expects": {
+      "@type": "@vocab"
```

Comment:
It's a bit annoying and error prone to have to declare this `@type` here. Perhaps we should refer to a hydra base context that already does this?

Reply:
Same for the other two predicates below

Reply:
I agree. If hydras' context already does it, let's not be redundant.

Reply:
That's a good point indeed @tpluscode

Reply:
Normally, I'd agree, but... The hydra vocabulary is a low level vocab, thus I don't expect we'll make ane cosmetic hallchanges, thus I'd rely on fact hydra based terms will be, well, hydra prefixed.

Reply:
Very well. I suppose that we can make the assumption since we also maintain the hydra context.

Could be a good API validation check to see that object of those props which should be `"@type": "@vocab"` (or "`"@type": "@id"`) properties are indeed resources and not accidental literals.

Reply:
So we agree to remove `@type: @vocab` and recommend that Hydra context is used?

Reply:
Yep, I'd go that way

Reply:
Here's an issue for the validator: hypermedia-app/hydra-validator#2

Reply:
I use the provided context now